portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -31,9 +31,6 @@
   @Override
   public void init() {
     messages = new ArrayList<String>();
```

Comment:
This can be part of the variable initialization now, consider inlining and dropping the init function.